observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemRegistrar.java

```diff
@@ -70,19 +83,21 @@ class MicrometerSubsystemRegistrar implements SubsystemResourceDefinitionRegistr
             "io.micrometer"
     };
 
+    @Deprecated
     public static final SimpleAttributeDefinition ENDPOINT = SimpleAttributeDefinitionBuilder
             .create(MicrometerConfigurationConstants.ENDPOINT, ModelType.STRING)
-            .setAttributeGroup(MicrometerConfigurationConstants.OTLP_REGISTRY)
             .setRequired(false)
+            .addFlag(AttributeAccess.Flag.ALIAS)
             .setAllowExpression(true)
             .setRestartAllServices()
             .build();
 
+    @Deprecated
     public static final SimpleAttributeDefinition STEP = SimpleAttributeDefinitionBuilder
             .create(MicrometerConfigurationConstants.STEP, ModelType.LONG, true)
-            .setAttributeGroup(MicrometerConfigurationConstants.OTLP_REGISTRY)
```

Comment:
This is more for a model expert as I don't recall, but is removing an attribute from an attribute group a breaking change? @pferraro @yersan

Reply:
I think the changes made to these AttributeDefinitions were ill-informed, so I've reverted them.